marlowe-dashboard-client/src/Capability/Marlowe.purs

```diff
@@ -418,7 +410,7 @@ instance monadMarloweAppM :: ManageMarlowe AppM where
   unsubscribeFromPlutusApp dataProvider plutusAppId = Websocket.unsubscribeFromContract $ toBack plutusAppId
   unsubscribeFromWallet dataProvider wallet = Websocket.unsubscribeFromWallet $ toBack wallet
 
-instance monadMarloweHalogenM :: (ManageMarlowe m, ManageWebsocket m) => ManageMarlowe (HalogenM state action slots Msg m) where
+instance monadMarloweHalogenM :: (ManageMarlowe m, ManageWebsocket m, MonadAsk Env m) => ManageMarlowe (HalogenM state action slots Msg m) where
```

Comment:
I was going to ask what this was for, but I think I can guess you had a good idea here, and forgot to follow through - I'm going to go ahead and push it as another commit (also as an excuse to restart hydra, because I suspect is failing for no good reason).